app/helpers/application_helper/button/csv.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::Csv < ApplicationHelper::Button::Basic
+  def check_rbac?
+    %w(render_report_csv chargeback_download_csv compare_download_csv).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
As I commented elsewhere these missing RBAC features will get added. There will be no list of features in this class.